<launch>
    <include file="$(find rto_bringup_sim)/launch/robot.launch" />

    <node pkg="final_project" type="reinforcmentLearning.py" name="final_project" output="screen">
        <param name="learnRate" value="0.9" />
        <param name="epsilon" value="0.9" />
        <param name="gamma" value="0.9" />
        <param name="maxEpisodes" value="3000" />
        <param name="epsilonDecrease" value="0.9987" />
        <param name="load" value="False" />
    </node>
</launch>